cram_pr2/cram_pr2_shopping_demo/src/plans.lisp

```diff
@@ -0,0 +1,173 @@
+(in-package :cram-pr2-shopping-demo)
+
+(defparameter *table* (cl-transforms-stamped:make-pose-stamped
+                        "map" 0.0
+                        (cl-transforms:make-3d-vector -3.1 0.7 0.75)
+                        (cl-transforms:make-identity-rotation)))
+
+(defparameter *pose-grasping* (cl-transforms-stamped:make-pose-stamped
+                               "map" 0.0
+                               (cl-transforms:make-3d-vector -1.5 -0.3 0)
+                               (cl-transforms:axis-angle->quaternion
+                                (cl-transforms:make-3d-vector 0 0 1) (/ pi -2))))
+
+(defparameter *pose-grasping-left* (cl-transforms-stamped:make-pose-stamped
+                               "map" 0.0
+                               (cl-transforms:make-3d-vector -1 -0.35 0)
+                               (cl-transforms:axis-angle->quaternion
+                                (cl-transforms:make-3d-vector 0 0 1) (/ pi -2))))
+
+(defparameter *pose-grasping-right* (cl-transforms-stamped:make-pose-stamped
+                               "map" 0.0
+                               (cl-transforms:make-3d-vector -2 -0.35 0)
+                               (cl-transforms:axis-angle->quaternion
+                                (cl-transforms:make-3d-vector 0 0 1) (/ pi -2))))
+
+
+(defparameter *pose-near-table* (cl-transforms-stamped:make-pose-stamped
+                                  "map" 0.0
+                                  (cl-transforms:make-3d-vector -2.5 0.5 0)
+                                  (cl-transforms:axis-angle->quaternion
+                                   (cl-transforms:make-3d-vector 0 0 1) (/ pi 1))))
+
+(defparameter *looking-pose-mid* (cl-transforms-stamped:make-pose-stamped
+                                  "map" 0.0
+                                  (cl-transforms:make-3d-vector -1.5 -1.05 1.15)
+                                  (cl-transforms:make-identity-rotation)))
+
+(defparameter *pose-detecting* (cl-transforms-stamped:make-pose-stamped
+                               "map" 0.0
+                               (cl-transforms:make-3d-vector -1.5 1 0)
+                               (cl-transforms:axis-angle->quaternion
+                                (cl-transforms:make-3d-vector 0 0 1) (/ pi -2))))
+
+(defun move-object (?object)
+    (let ((?grasping-arm :right)
+          (?grasp-pose *pose-grasping*)
+          (?grasp-pose-left *pose-grasping-left*)
+          (?grasp-pose-right *pose-grasping-right*)
+          (?pose-near-table *pose-near-table*)
+          (?table *table*)
+          (map->obj NIL)
```

Comment:
When your starting value for a variable is NIL you can simply not assign it, as that is the default value of all variables in the LET block:
```lisp
(let ((?table *table*)
       map->obj
       map->base
       trans) ....)
```